### PR TITLE
US129560 [AQ] Add Quiz Detail Table Column Config Support to Settings Page - UI

### PR DESCRIPTION
**Pull request by Vitalii-Misechko**
[US129560](https://rally1.rallydev.com/#/?detail=/userstory/602834337163&fdp=true): [AQ] Add Quiz Detail Table Column Config Support to Settings Page - UI

* Functional Testing 
  * [x] Browsers (Chrome, FF, Edge)
  * [x] Accessibility
  * [x] RTL
  * [x] Responsive
  * [x] Test cases
* Security, devops, perf: N/A
* UX and docs
  * [x] Demo to Kevin
  
FYI @Brightspace/draco-vis

### Screenshot

image

### CODE REVIEW

File: locales/en.js [hunk 1]
```diff
@@ -92,13 +92,19 @@ export default {
...
 	"settings:quizReliabilityCardTitle": "Quiz Reliability",
 	"settings:quizReliabilityCardDescription": "The Quiz Reliability card shows the distribution of quizzes across low, moderate and high reliability. All quizzes with a negative reliability coefficient will populate as an outlier. The ranges can be defined in Reliability Coefficient. The card will be displayed in the Summary section.",
 
+	"settings:quizReliabilityColumnTitle": "Reliability Coefficient",
+	"settings:quizReliabilityColumnDescription": "The Reliability Coefficient is calculated to show the consistency of the quiz results. Reliability will usually range from 0 to 1 with negative values being possible. The data point will display in the Quiz Detail Table.",
+	"settings:flagQuizReliabilityEditLabel": "Edit reliability coefficient threshold! TODO!",
+	"settings:highCoefQuizReliabilityEditDescription": "High coefficients are defined as above",
+	"settings:highCoefQuizReliabilityEditLabel": "Edit high threshold for reliability coefficient! TODO!",
```
Comment (Vitalii-Misechko) [inline locales/en.js:106]: working on TODOs

File: assessment-quality-dashboard.js [hunk 2]
```diff
@@ -31,7 +31,10 @@ class AssessmentQualityDashboard extends Localizer(MobxLitElement) {
 			showAverageScoreCol: { type: Boolean, attribute: 'show-average-score-col', reflect: true },
 			averageScoreFlagThreshold: { type: Number, attribute: 'average-score-flag-threshold', reflect: true },
 			showAverageScoreCard: { type: Boolean, attribute: 'show-average-score-card', reflect: true },
-			showQuizReliabilityCard: { type: Boolean, attribute: 'show-quiz-reliability-card', reflect: true }
+			showQuizReliabilityCol: { type: Boolean, attribute: 'show-quiz-reliability-col', reflect: true },
+			showQuizReliabilityCard: { type: Boolean, attribute: 'show-quiz-reliability-card', reflect: true },
+			quizReliabilityFlagThreshold: { type: Number, attribute: 'quiz-reliability-flag-threshold', reflect: true },
+			quizReliabilityHighCoefThreshold: { type: Number, attribute: 'quiz-reliability-high-coef-threshold', reflect: true }
```
[the remark this answers is not in the document]
Comment (bemailloux) [reply assessment-quality-dashboard.js:37]: Also, I haven't looked at the validation story yet but does the following relationship have to be true? `highThreshold >= flagThreshold`?

or is `highThreshold < flagThreshold` also acceptable?

If `highThreshold < flagThreshold` is possible then my name suggestion doesn't make sense

[the remark this answers is not in the document]
Comment (Vitalii-Misechko) [reply assessment-quality-dashboard.js:37]: [US130517](https://rally1.rallydev.com/#/?detail=/userstory/605755048093&fdp=true): _[AQ] Settings Page Form Validation_
covers relation between `highThreshold`  and `flagThreshold`. Basically, the story establishes `highThreshold >  flagThreshold`

[the remark this answers is not in the document]
Comment (bemailloux) [reply assessment-quality-dashboard.js:37]: Maybe `quizReliabilityFlagThreshold` and `quizReliabilityHighThreshold` then?

**Review by bemailloux — APPROVED** [review]

File: test/components/settings/quiz-details-settings-list.test.js [hunk 3]
```diff
@@ -110,6 +131,30 @@ describe('d2l-insights-quiz-details-settings-list', () => {
...
 		});
+
+		it('should update settings value when Reliability Coefficient flag threshold input is modified', async() => {
+			const input = el.shadowRoot.querySelector('#quiz-reliability-threshold-edit');
+			await input.updateComplete;
+
+			// hacky way to "pretend" the input value changed
+			input.value = 0.323;
+			input.dispatchEvent(new Event('change'));
+			await input.updateComplete;
```
Comment (NicholasHallman) [inline test/components/settings/quiz-details-settings-list.test.js:142]: Clever!

**Review by NicholasHallman — APPROVED** [review]

File: assessment-quality-dashboard.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (Vitalii-Misechko) [reply assessment-quality-dashboard.js:37]: It works for me. I'll update these

File: test/components/settings/quiz-details-settings-list.test.js (hunk 3 above)
Comment (Vitalii-Misechko) [reply test/components/settings/quiz-details-settings-list.test.js:142]: All credits to @bemailloux

File: assessment-quality-dashboard.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (Vitalii-Misechko) [reply assessment-quality-dashboard.js]: > Maybe `quizReliabilityFlagThreshold` and `quizReliabilityHighThreshold` then?

done

**Pull request merged**